api/envoy/config/rbac/v2alpha/rbac.proto

```diff
@@ -68,90 +74,70 @@ message Policy {
   repeated Principal principals = 2 [(validate.rules).repeated .min_items = 1];
 }
 
-// Specifies how to match an entry in a map.
-message MapEntryMatch {
-  // The key to select an entry from the map.
-  string key = 1;
+// Permission defines a permission to access the service.
+message Permission {
 
-  // A list of matched values.
-  repeated envoy.type.StringMatch values = 2;
-}
+  message Set {
+    repeated Permission rules = 1 [(validate.rules).repeated .min_items = 1];
```

Comment:
Per offline convo can we add some more docs/explanation of what the AND/OR behavior is of this repeated lists? I think that would be helpful.

Reply:
Ah, these are just wrappers since you can't have a repeated field in a oneof. Adding a comment to specify that the application is dependent on whether it's in `and_rules/ids` vs `or_rules/ids`

Reply:
Sorry, I commented in the wrong place. :(

I meant here: https://github.com/rodaine/envoy/blob/c18561dfce1acc3e9cd101149f21b786a8384626/api/envoy/config/rbac/v2alpha/rbac.proto#L71-L74

Reply:
(But more comments here fine too)